keybert/llm/_litellm.py

```diff
@@ -68,6 +68,7 @@ class LiteLLM(BaseLLM):
     def __init__(self,
                  model: str = "gpt-3.5-turbo",
                  prompt: str = None,
+                 system_content: str = "You are a helpful assistant.",
```

Comment:
Could you change the naming from `system_content`, to `system_prompt`? That is more in line with the `prompt` parameter that already exists. Also, could you add a description of what this parameter does to the docstrings?